src/blog/2024/01/dashboard-2-ga.md

```diff
@@ -0,0 +1,46 @@
+---
+title: Dashboard 2.0 is Generally Available!
+subtitle: This week sees our first major version release of Node-RED Dashboard 2.0!
+description: This week sees our first major version release of Node-RED Dashboard 2.0!
+date: 2024-01-16
+authors: ["joe-pavitt"]
+image: "/blog/2023/12/images/tile-dashboard-0-10-0.png"
+tags:
```

Comment:
If you look at the `relatedPosts` filter now in the `main` branch, this way you end up in many categories, but never as a related post. And now I'm not sure what the best way is to approach this. (cc @Yndira-FlowForge )

Reply:
The related/recommend articles just takes from the most recently published articles?

Reply:
Read the code again :) Yndira iterated a bit

Reply:
Currently, it takes the most recent articles that have the same set of tags.

I did [this](https://github.com/FlowFuse/website/pull/1544/files#r1457634202) too, but I'm not so sure about it now. WDYT?